MITechnologies/Formitize-NET-API
# Develop steve review

### Task:
https://atebion.monday.com/boards/4011769748/pulses/4167928065
https://atebion.monday.com/boards/4011769748/pulses/4167928735

#### Changes made
* Updated API docs to match current functionality
* Added a "Run in Postman" button to make it easy for people to fork the collection. 

#### Screenshot(s)
![image](https://user-images.githubusercontent.com/7939537/227113104-ea988fe8-ea24-4143-89bd-323df606e050.png)
![image](https://user-images.githubusercontent.com/7939537/227108455-4a2cf732-ad68-4036-a0e1-56450e2d00d6.png)


#### How to test
1. Go to the newly updated API docs and press the "Run in Postman" button to test. 

See also: https://github.com/MITechnologies/formitize-backend/pull/1308
